gui/mozregui/ui/intro.ui

```diff
@@ -79,6 +79,26 @@
     </widget>
    </item>
    <item row="5" column="0">
+    <widget class="QLabel" name="lang_label">
+      <property name="toolTip">
+        <string>enter the code of the lang you want a build for</string>
+      </property>
+      <property name="text">
+        <string>Lang</string>
+      </property>
+    </widget>
+   </item>
+   <item row="5" column="1">
+     <widget class="QLineEdit" name="lang">
+       <property name="toolTip">
+        <string>Lang code such as he, ar, zh-TW</string>
+      </property>
+      <property name="placeholderText">
+        <string>The lang code</string>
```

Comment:
```suggestion
        <string>The lang code (e.x. de)</string>
```

Reply:
Thanks, done (I think I thought about when writing it, but decided to match Repository by only having it in the title).